### DISCUSSION TITLE
Exporting anisotropy matrix

### DISCUSSION THREAD

**Discussion by savvy73** [simnibs/simnibs#501] [6. Getting started] [answered]
Hi, I have been trying to get the anisotropy values of GM and WM from the example dataset (ernie/m2m_ernie). I ran dwi2 cond to get the diffusion tensors from GM and WM. How can I get the anisotropy? Should I be using cond.py ?
Thank you in advance.

**Comment by axelthi** [comment, marked as answer]
Hi, it is likely easiest when you modify the SimNIBS code to export the tensors. Please insert the following code in simnibs.simulation.sim_struct.py after line 1684:
```
        if self.anisotropy_type != 'scalar':           
            import nibabel as nib
            tensor_nifti = nib.load(self.fn_tensor_nifti)
            img = cond.interpolate_to_grid(tensor_nifti.shape[:3], 
                                            tensor_nifti.affine, 
                                            method='linear', 
                                            continuous=False)
            img = img[:,:,:,[0,1,2,4,5,8]]
            tmp_nii = nib.Nifti1Image(img, tensor_nifti.affine)
            nib.save(tmp_nii, fn_no_extension+'_CTI.nii.gz')
```

Then set up a tDCS simulation with anisotropic conductivities. You will get the conductivity tensor image in the simulation results folder.
